openwisp_radius/private_storage/views.py

```
@@ -19,3 +23,24 @@ def can_access_file(self, private_file):
+        operation_description=_(
+            "Allows downloading the CSV file used to import users for a"
+            "specific batch user creation operation."
+        ),
+        responses={
+            200: openapi.Response(
+                description=_("CSV file"), schema=openapi.Schema(type=openapi.TYPE_FILE)
+            ),
+        },
+        tags=["radius"],
+    )
+    def get(self, request, slug, pk, *args, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `slug`

(ARG002)